pkg/apiserver/slowquery/service.go

```diff
@@ -99,6 +99,14 @@ func (s *Service) getDetails(c *gin.Context) {
 		rest.Error(c, err)
 		return
 	}
+
+	// get visual plan
+	result.VisualPlan, err = utils.GenerateVisualPlanJSON(result.VisualPlan)
+	if err != nil {
+		rest.Error(c, err)
+		return
+	}
```

Comment:
Will this work with v6.1?

Reply:
No, it's merged into the feature branch.

Reply:
Does this mean that we need to cherry pick instead of fast-forward when publishing new releases?

Reply:
It will be horrible if the master branch of TiDB Dashboard does not work with the master branch of TiDB.

Reply:
Oh, I got you. This PR is not merging into master.

Reply:
Though the `FeatVisualPlan` branch is designed for collaboration of frontend and backend and better review experiences.

Maybe cherry pick for the specific release branches will be better in the future.

Reply:
> Oh, I got you. This PR is not merging into master.

yes~